startup_runners/startup_runner.c

```diff
@@ -25,6 +25,10 @@ main(int argc, char **argv)
 {
     double    start_time = -1;
 
+    /* silence warnings */
+    argc = argc;
+    argv = argv;
```

Comment:
I assume that this program doesn't use `argc` or `argv` in which case it would be better to use `int main(void)` as the declaration.